### PR TITLE
Update Appveyor config.

### PR DESCRIPTION
**Pull request by ADmad**
Let's see if this prevents appveyor builds from breaking ever so often.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -14,40 +17,50 @@ branches:
...
+  - IF %PHP%==1 echo extension=php_mbstring.dll >> php.ini
+  - IF %PHP%==1 echo extension=php_intl.dll >> php.ini
+  - IF %PHP%==1 echo extension=php_fileinfo.dll >> php.ini
+
+  - IF %PHP%==1 curl -fsS https://windows.php.net/downloads/pecl/releases/pdo_sqlsrv/5.2.0/php_pdo_sqlsrv-5.2.0-7.2-nts-vc15-x64.zip -o pdosqlsrv.zip
+  - IF %PHP%==1 7z x pdosqlsrv.zip -oC:\php\ext php_pdo_sqlsrv.dll > nul
+  - IF %PHP%==1 curl -fsS https://windows.php.net/downloads/pecl/releases/sqlsrv/5.2.0/php_sqlsrv-5.2.0-7.2-nts-vc15-x64.zip -o sqlsrv.zip
+  - IF %PHP%==1 7z x sqlsrv.zip -oC:\php\ext php_sqlsrv.dll > nul
+  - IF %PHP%==1 echo extension=php_pdo_sqlsrv.dll >> php.ini
+  - IF %PHP%==1 echo extension=php_sqlsrv.dll >> php.ini
```
Comment (markstory) [inline appveyor.yml]: Doesn't seem like these extensions are getting downloaded/installed :cry:

**Comment by ADmad** [comment]
Don't know why the session tests fail on Appveyor/windoz :(

**Comment by ADmad** [comment]
Thanks @WyriHaximus for this blog post https://blog.wyrihaximus.net/2016/11/running-php-unit-tests-on-windows-using-appveyor-and-chocolatey/

File: appveyor.yml (hunk 1 above)
Comment (ADmad) [reply appveyor.yml]: They weren't being loaded due to PHP version mismatch. Fixed now.

**Comment by WyriHaximus** [comment]
No problem 👍 . Also 3 failures out nearly 10K, not bad!

**Comment by markstory** [comment]
Might be worth dumping out what is going on with binary data. I know that the BinaryType has some SQLServer specific logic in it.

**Comment by markstory** [comment]
Nice work @ADmad :clap:

**Pull request merged**